Documentation/Columns/Properties/DisplayCond.rst

```diff
@@ -126,6 +126,20 @@ Multiple conditions can be combined::
    ],
 
 
+An example with multiple values and :code:`OR`::
```

Comment:
```suggestion
An example with multiple values and :code:`OR`:

..  code-block:: php
   :caption: EXT:my_extension/Configuration/TCA/Overrides/tx_mask_field.php
```
Thanks a lot for your contribution! Just some small changes needed:

We always use verbose Code-Block with a caption stating where the Code should be put as this topic tends to confuse newbies.